### ISSUE TITLE
About calling compartment PC1 values

### ISSUE THREAD

**Issue by GlenZhang-GZ** [robinweide/GENOVA#327]
Can GENOVA call compartment PC1 values by gene density or GC content? If I don‘t have an activate histone marker (peak file), how can I get PC1 values by GENOVA? Thanks!

**Comment by teunbrand** [comment]
Yes this is documented in `?sign_compartmentscore`. For GC content, you can use a bedgraph and for gene density you can use a bed file of TSSs.
